### DISCUSSION TITLE
Any luck on installing NeXTSTEP on v86?

### DISCUSSION THREAD

**Discussion by Alsefina** [copy/v86#933] [Q&A] [answered]
Im trying to run NeXTSTEP 3.3 on v86. Setup starts without issue, however I seem to get stuck when it asks me which SCSI interface the CD drive is connected to. So if anyone has ran NeXTSTEP on v86 in the past, whats the procedure for doing so? 

my emulation config:

CD: NeXTSTEP 3.3 ISO (m68k & i386)
Floppy: i386 Boot image for nextstep
HDD: 1GB qcow2 disk I made with qemu
RAM: 64M

Also, is there any way to get more virtual floppy drives? Im wondering if the setup is hanging because I had to swap out boot disk for the driver disk during install.

**Comment by SuperMaxusa** [comment, marked as answer]
> So if anyone has ran NeXTSTEP on v86 in the past, whats the procedure for doing so?

I tried installing NextSTEP 3.3 and OpenSTEP 4.2, but no luck. 
About NextSTEP 3.3, while [installation](https://gunkies.org/wiki/Installing_NeXTSTEP_on_Qemu) I choosed `Primary/Secondary(Dual) EIDE and ATAPI Device Controllers (v3.35)` (can be found on beta driver floppy). On starting system, I get cyclic error like `hc0: ATA command c4 failed. Retrying...` (and some errors about Floppy controller, but I disabled it)

![ns 3.3](https://github.com/copy/v86/assets/41739128/f9b9d773-363b-4e85-aeac-8ed3fb088df4)

However, there is a problem with *Step that they cannot be installed on new versions of QEMU (https://www.emaculation.com/forum/viewtopic.php?t=11469), so it will be difficult to debug :(

**Reply by Alsefina** [reply]
> they cannot be installed on new versions of QEMU 

what's the last version of QEMU that STEP worked on?

**Reply by SuperMaxusa** [reply]
personally I tested only QEMU 0.9 (see https://virtuallyfun.com/2017/04/22/darwin-0-1-rhapsody-dr-2-booted/ and https://gunkies.org/wiki/Installing_NeXTSTEP_on_Qemu#Configuring_Qemu)

UPD: I found patch for QEMU 6.x, that fixes bug similar below: https://www.nextcomputers.org/forums/index.php?PHPSESSID=6e7144e491e68da54aef4cc22505c437&msg=27804

**Reply by SuperMaxusa** [reply]
UPD: I have installed Apple Rhapsody RP2 with enabled `EIDE and ATAPI Device Controller (v5.0.1)`.
Seems that error `ATA command c4 failed.` doesn't appear, dut now it stucks after `Power management is enabled`.

![applerhapsody](https://github.com/copy/v86/assets/41739128/f6829119-192d-40a9-ae8a-bc25d558a3a8)

I think that better create the issue about starting Darwin (NextStep, Rhapsody, OpenStep and etc)
